src/devices/Bmx280/BmxBase.cs

```diff
@@ -53,6 +46,7 @@ private void Begin()
             _calibrationData.ReadFromDevice(this);
         }
 
+
         /// <summary>
         /// Sets the power mode to the given mode
         /// </summary>
```

Comment:
I think we should use property to read data and set settings, and change Task.Delay() to DelayHelper.DelayMilliseconds().
https://github.com/dotnet/iot/blob/3c55f83a2fd60557d1373ca126abcacfaaeea9de/src/devices/Common/System/Device/DelayHelper.cs#L13